doc/ChangeLog

```diff
@@ -1,3 +1,59 @@
+
+===============================================================
+
+Tag name:
+Originator(s): cacraig
+Date: Dec 24, 2024
+One-line Summary: Complete Zhang McFarlane conversion to CCPP
+Github PR URL:
```

Comment:
Just adding a reminder to finish the `ChangeLog` or add the same info to the PR description before merging.